core/spec/models/spree/payment_create_spec.rb

```diff
@@ -81,6 +81,10 @@ module Spree
         }
       end
 
+      around do |example|
+        ActiveSupport::Deprecation.silence { example.run }
```

Comment:
We don't want to silence `ActiveSupport::Deprecation` but `Spree::Deprecation` here